enterprise/signals.py

```diff
@@ -63,27 +64,21 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
     if user_instance is None:
         return  # should never happen, but better safe than 500 error
 
-    try:
-        pending_ecu = PendingEnterpriseCustomerUser.objects.get(user_email=user_instance.email)
-    except PendingEnterpriseCustomerUser.DoesNotExist:
-        pending_ecu = None
+    pending_ecus = PendingEnterpriseCustomerUser.objects.filter(user_email=user_instance.email)
```

Comment:
If we’re only looking up by email, no need to index together with customer. I misunderstood that we’d be using both in the lookup

Reply:
Most other places where we call `PendingEnterpriseCustomerUser.objects.get` already supplied both a `user_email` and an `enterprise_customer`. Would it make sense to keep the index together with customer for those cases?

Reply:
Yep that makes sense. You can also add user email on it’s own for this case like in the docs example:

class Meta:
        indexes = [
            models.Index(fields=['last_name', 'first_name']),
            models.Index(fields=['first_name'], name='first_name_idx'),
        ]

Reply:
Ah, got it. Cool, will update!